src/Namespaces/OrderInvoicesNamespace.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Hitmeister\Component\Api\Namespaces;
+
+use Hitmeister\Component\Api\Cursor;
+use Hitmeister\Component\Api\Endpoints\OrderInvoices\Delete;
+use Hitmeister\Component\Api\Endpoints\OrderInvoices\Find;
+use Hitmeister\Component\Api\Endpoints\OrderInvoices\Get;
+use Hitmeister\Component\Api\Endpoints\OrderInvoices\Post;
+use Hitmeister\Component\Api\Exceptions\ResourceNotFoundException;
+use Hitmeister\Component\Api\Exceptions\ApiException;
+use Hitmeister\Component\Api\FindBuilder;
+use Hitmeister\Component\Api\Helper\Response;
+use Hitmeister\Component\Api\Namespaces\Traits\PerformWithId;
+use Hitmeister\Component\Api\Transfers\OrderInvoiceAddTransfer;
+use Hitmeister\Component\Api\Transfers\OrderInvoiceListTransfer;
+use Hitmeister\Component\Api\Transfers\OrderInvoiceTransfer;
+
+class OrderInvoicesNamespace extends AbstractNamespace
+{
+	use PerformWithId;
+
+	/**
+	 * Find all invoices paginated via limit and offset.
+	 *
+	 * @param int $limit
+	 * @param int $offset
+	 *
+	 * @return Cursor|OrderInvoiceListTransfer[]
```

Comment:
Changes on the server side to the OrderInvoiceList -> OrderInvoice transfer object need to be applied here.